source/includes/reference/_webhooks.md

```diff
@@ -1,6 +1,5 @@
-# Webhooks
-
-**NB! Please note this version of webhooks is now **_deprecated_**. You can use (profile webhooks) or (application webhooks) instead.
+# Webhooks (v1)
+#### Please note this version of webhooks is now **_deprecated_**. You can use (profile webhooks) or (application webhooks) instead.
```

Comment:
>You can use (profile webhooks) or (application webhooks)

Possible to add anchors to both of those to jump up to the relevant webhook sections? The brackets could be removed then.

Reply:
Good catch, I meant to add links here 😅